docs/pages/beta/crm-beta/about.mdx

```diff
@@ -11,7 +11,7 @@ We're working day and night to finish this migration, and making steady progress
 
 In the meantime, **please bookmark this page** as it's unreachable from the main documentation page.
 
-Additionally, you may notice there is a "CRM API" anchor in the sidebar. The documentation you find there **is for the old CRM API** which requires a separate API key and set up process. If you find functionality on those documentation pages that you need supported in the new CRM API, please reach out to your engineering point of contact.
+Additionally, you may notice there is a "CRM API" anchor in the sidebar. The documentation you find there **is for the old CRM API** which requires a separate API Token and set up process. If you find functionality on those documentation pages that you need supported in the new CRM API, please reach out to your engineering point of contact.
```

Comment:
Starting to get more people onboard to the new product. We should finalize how we want to cut up the docs between the Passthrough and ETL APIs by the end of this week. Want to sync midweek after you had sometime to think about it or would it be too rushed?